packages/calcite-ui-icons/docs/keywords.json

```diff
@@ -8930,5 +8930,19 @@
     ],
     "category": "Lists",
     "release": "3.29.6"
+  }, 
+  "web-adaptor":{
+    "alias":[
+      "3.29.7",
+      "gis",
+      "web", 
+      "diagram", 
+      "nodes", 
+      "storage", 
+      "servers", 
+      "transfer"
+    ],
+    "category":"GIS",
+    "release":"3.29.7"
```

Comment:
This version is incorrect, please see my comment in the other PR - https://github.com/Esri/calcite-design-system/pull/10278#discussion_r1755803192